src/transformers/models/regnet/modeling_tf_regnet.py

```diff
@@ -447,11 +447,11 @@ def __init__(self, config: RegNetConfig, *inputs, **kwargs):
     )
     def call(
         self,
-        pixel_values: tf.Tensor = None,
-        labels: tf.Tensor = None,
-        output_hidden_states: bool = None,
-        return_dict: bool = None,
-        training=False,
+        pixel_values: Optional[tf.Tensor] = None,
+        labels: Optional[tf.Tensor] = None,
+        output_hidden_states: Optional[bool] = None,
+        return_dict: Optional[bool] = None,
```

Comment:
Maybe do `tf.Tensor | None` here to match the rest?

Reply:
Ah, I just saw your comment above! Our CI runner should be able to handle `| None` - I wonder what went wrong?

Reply:
Yeah, I find it quite uncanny, as the other annotations are not complaining :thinking: 

I'm happy to further investigate this, are you happy for me to open a follow up issue, so we can close this dreary type hints issue, or do you think that there are more pressing stuff to be solved before (the list of issues is already huge)?

Reply:
I don't think there's any real need here, since both kinds of type hints are fine!